### PR TITLE
feat: Add new Etherscan icons and update Etherscan link in AccountDashboard [2/x] JB-65

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

This PR introduces two new Etherscan icons (etherscan_od.svg and etherscan_ol.svg) and updates the Etherscan link in the AccountDashboard component to use the new Etherscan icon.

Changes made in this PR:
1. Add new Etherscan icons (etherscan_od.svg and etherscan_ol.svg) to the public/assets/icons folder.
2. Update the AccountDashboard component to use the new Etherscan icons and the SocialButton component.

The new Etherscan icons provide a better visual representation and the updated link in the AccountDashboard component improves the overall user experience.

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] (if relevant) I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] (if relevant) I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3327** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3327" target="_blank">Graphite</a> 
    * **PR #3328** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3328" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3328?utm_source=stack-comment).

File: src/components/icons/Etherscan.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <img
+      src={forThemeOption?.({
+        dark: '/assets/icons/etherscan_od.svg',
+        light: '/assets/icons/etherscan_ol.svg',
+      })}
+      alt="Etherscan logo"
+      width={size || 15}
+      height={size || 15}
+    />
```
Comment (tomquirk) [inline src/components/icons/Etherscan.tsx:15]: **suggestion** should be able to use `next/image` for this

Comment (wraeth-eth) [reply src/components/icons/Etherscan.tsx:15]: i am against using next image. It is a headache and the consensus from others is that its not worth using in the version of next we use. 

Apparently it's better in later versions

**Comment by wraeth-eth** [comment]
@wraeth-eth queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3328).

**Comment by wraeth-eth** [comment]
[Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3328) couldn't merge this PR because it had conflicts with the trunk branch.

**Comment by wraeth-eth** [comment]
@wraeth-eth queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3328).

**Comment by wraeth-eth** [comment]
@wraeth-eth merged this pull request with [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3328).

**Pull request merged**